cfme/tests/containers/test_containers_default_project_replicators.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import CheckboxTable
+from cfme.containers.replicator import Replicator
+from itertools import product
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        (lambda: current_version() < "5.6") or (lambda provider: provider.version < 3.2)),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope='function')
+
+list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
+
+
+REPLICATORS_PROPERTIES_FIELDS = ['Name', 'Creation timestamp', 'Resource version',
```

Comment:
Consider passing in IDs as well instead of passing only values.
You can turn this
```
test_containers_replicators_properties[cm-env2-replicators_properties_fields0]
test_containers_replicators_properties[cm-env2-replicators_properties_fields1]
test_containers_replicators_properties[cm-env2-replicators_properties_fields2]
```
into much more valuable
```
test_containers_replicators_properties[cm-env2-name]
test_containers_replicators_properties[cm-env2-creation_timestamp]
test_containers_replicators_properties[cm-env2-resource_version]
```
.
Next step would be to make the test names shorter and we can land at something like:
```
test_replicator_properties[cm-env2-name]
test_replicator_properties[cm-env2-creation_timestamp]
test_replicator_properties[cm-env2-resource_version]
```